app/models/menu_item.rb

```diff
@@ -0,0 +1,5 @@
+class MenuItem < ApplicationRecord
+    validates :name, presence: true
```

Comment:
please refactor this

E.g
`validates_presence_of :name, :description, :price`

Reply:
Refactored with modern syntax.